cypress-tests/cypress/e2e/rstudioSession.cy.ts

```diff
@@ -0,0 +1,112 @@
+import { rstudioTestFuncs } from "@renku/notebooks-cypress-tests";
+import { v4 as uuidv4 } from 'uuid';
+import { registerCustomCommands } from "../support";
+
+const email = Cypress.env("TEST_EMAIL")
+const username = Cypress.env("TEST_USERNAME")
+const password = Cypress.env("TEST_PASSWORD")
+const firstName = Cypress.env("TEST_FIRST_NAME")
+const lastName = Cypress.env("TEST_LAST_NAME")
+const projectName = `test-project-${uuidv4()}`
+
+describe('Basic rstudio functionality', () => {
```

Comment:
This a minor detail, but would you mind switching all the `'` to `"` just for coherence? Some in a few other place on this file.
At some point we could add a linter (eslint?) for this and other details, but it's definitely not urgent nor necessary in this PR.